pages/pipelines/build_matrix.md.erb

```diff
@@ -1,6 +1,14 @@
 # Build Matrix
 
-Build Matrices lets you simplify complex build configurations by expanding a step template and array of matrix elements into multiple jobs.
+Build Matrices help you simplify complex build configurations by expanding a step template and array of matrix elements into multiple jobs.
+
+The following [command step](/docs/pipelines/command-step) attributes can contain matrix values for interpolation:
+
+* [environment variables](/docs/pipelines/environment-variables)
+* [labels](/docs/pipelines/command-step#label)
+* [commands](/docs/pipelines/command-step#command-step-attributes)
+
+Interpolation of matrix values inside other attributes, including step keys and plugin attributes is not supported.
```

Comment:
Feels like the most important thing here (the negation) is at the very, very end. Maybe flip it, like this?

```suggestion
You can't use matrix values in other attributes, including step keys and plugin attributes.
```

Reply:
LOL, can't believe I wrote that.